zanata-war/src/main/java/org/zanata/action/validator/DuplicateEmailValidator.java

```diff
@@ -34,15 +34,20 @@
 
     @Override
     public boolean isValid(String string, ConstraintValidatorContext context) {
-        if (string == null)
+        if (string == null) {
             return true;
-        if (string.length() == 0)
+        }
+        if (string.length() == 0) {
             return true;
-        PersonDAO personDAO =
-                ServiceLocator.instance().getInstance(PersonDAO.class);
+        }
+        PersonDAO personDAO = getPersonDAO();
```

Comment:
I know it's not substantial; that's why it should be separated from the substantial changes.